src/WordPress/Hooks.php

```diff
@@ -256,6 +256,20 @@ public function getPostRelatedLinks($postId)
             array_push($listofurls, $pageLink);
         }
 
+        // Refresh pagination
+        $total_posts_count = wp_count_posts()->publish;
+        $posts_per_page = get_option('posts_per_page');
+        // Limit to up to 3 pages
+        $page_number_max = min(3, ceil($total_posts_count / $posts_per_page));
```

Comment:
i wonder if we should make this configurable for those who want to do say 2 or 10 instead of the static 3?

Reply:
We can do this once requested by customers. I think it's ok to ship without configuration option.